src/Codeception/Lib/Parser.php

```diff
@@ -125,9 +124,33 @@ protected function addCommentStep($comment)
         $this->scenario->addStep(new \Codeception\Step\Comment($comment, []));
     }
 
+    public static function validate($file)
+    {
+        if (defined('HHVM_VERSION') // not for HHVM
+            || (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') // not for WIN
```

Comment:
Yes, it works but Appveyor fails there. Somehow it doesn't handle well the error from another process
https://ci.appveyor.com/project/DavertMik/codeception/build/1.0.1020#L504